Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -76,30 +182,44 @@ public final class System {
     /// - Returns: The default AddressScheme
     ///
     public func defaultAddressScheme (network: Network) -> AddressScheme {
-        switch network.currency.code {
-        case Currency.codeAsBTC: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_BTC_SEGWIT)
-        case Currency.codeAsBCH: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_BTC_LEGACY)
-        case Currency.codeAsETH: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_ETH_DEFAULT)
-        default: return AddressScheme (core: CRYPTO_ADDRESS_SCHEME_GEN_DEFAULT)
-        }
+        return defaultAddressSchemeMap[network.uids] ?? .genDefault
     }
 
     ///
+    /// Wallet Manager Modes
+    ///
+
+    var supportedModesMap: [String:[WalletManagerMode]] = [
+        "bitcoin-mainnet":      [.p2p_only],
```

Comment:
Are we missing api_only for bitcoin?

Reply:
We only add 'api_only' when BDB returns block height other than '-1' (BDB supports the blockchain).